dvc/commands/queue/status.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import logging
+from typing import List, Mapping, Optional
+
+from dvc.cli.command import CmdBase
+from dvc.cli.utils import append_doc_link
+from dvc.compare import TabularData
+
+from ..experiments.show import format_time
```

Comment:
```suggestion
from dvc.experiments.show import format_time
```

Reply:
I think the original here was correct, the helper is from `commands.experiments.show` (and not `dvc.repo.experiments`).

If anything though, this kind of formatting method should maybe go into `dvc.cli.utils` instead?